### PR TITLE
WW-4998 I18nInterceptor's default storage should store locale

### PR DESCRIPTION
**Pull request by aleksandr-m**

### CODE REVIEW

File: core/src/main/java/org/apache/struts2/interceptor/I18nInterceptor.java [hunk 1]
```diff
@@ -281,15 +282,13 @@ public Locale find() {
 
         @Override
         public Locale store(ActionInvocation invocation, Locale locale) {
-            HttpSession session = ServletActionContext.getRequest().getSession(false);
+            Map<String, Object> session = invocation.getInvocationContext().getSession();
```
Comment (lukaszlenart) [inline core/src/main/java/org/apache/struts2/interceptor/I18nInterceptor.java]: Why not to call `HttpSession session = ServletActionContext.getRequest().getSession();` directly? The the `if` below can be dropped.

Comment (aleksandr-m) [reply core/src/main/java/org/apache/struts2/interceptor/I18nInterceptor.java]: I think the `invocation.getInvocationContext().getSession()` can be `null` in some cases, can't it? And we need it to put locale.

Comment (lukaszlenart) [reply core/src/main/java/org/apache/struts2/interceptor/I18nInterceptor.java]: ah, right :)

**Comment by lukaszlenart** [comment]
@apache/struts-committers if no objections, LGTM?

**Comment by yasserzamani** [comment]
LGTM too 👍

**Comment by lukaszlenart** [comment]
LGTM 👍

**Pull request merged**